easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.3.0-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,178 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+builddependencies = [
+    ('Bazel', '3.4.1'),
+    ('protobuf', '3.10.0'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.23.0', '-nodocs'),
+    ('pybind11', '2.4.3', versionsuffix),
+]
+dependencies = [
+    ('Python', '3.7.4'),
+    ('h5py', '2.10.0', versionsuffix),
+    ('cuDNN', '7.6.4.38'),
+    ('NCCL', '2.4.8'),
+    ('cURL', '7.66.0'),
+    ('double-conversion', '3.1.4'),
+    ('flatbuffers', '1.12.0'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.0.3'),
+    ('ICU', '64.2'),
+    ('JsonCpp', '1.9.3'),
+    ('libjpeg-turbo', '2.0.3'),
+    ('LMDB', '0.9.24'),
+    ('NASM', '2.14.02'),
+    ('nsync', '1.24.0'),
+    ('SQLite', '3.29.0'),
+    ('PCRE', '8.43'),
+    ('protobuf-python', '3.10.0', versionsuffix),
+    ('libpng', '1.6.37'),
+    ('snappy', '1.1.7'),
+    ('SWIG', '4.0.1'),
+    ('zlib', '1.2.11'),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+use_pip = True
+
+# Dependecies created and updated using findPythonDeps.sh:
+# https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d
+exts_list = [
+    ('scipy', '1.4.1', {
```

Comment:
@Flamefire We should also remove this in favor of using `scipy` as a dependency?